timemachine/fe/interpolate.py

```diff
@@ -138,8 +138,8 @@ def align_chiral_bond_idxs_and_params(src_idxs, src_params, src_signs, dst_idxs,
             dst_idxs,
             zip(dst_signs, dst_params),
             make_default=lambda p: (p[0], 0),  # p[0] is sign, 0 is force constant
-            key=lambda idxs, p: (idxs, p[0]),  # use idxs and sign as key
-            get_idxs=lambda key: key[0],
+            key=lambda idxs, p: (tuple(idxs), p[0]),  # use idxs and sign as key
+            get_idxs=lambda key: tuple(key[0]),
```

Comment:
another (maybe nicer?) fix might be to change the definition of `to_hashable` to also convert nested arrays to tuples. This seems to work:
```python
    def to_hashable(x):
        return tuple(to_hashable(e) for e in x) if isinstance(x, Iterable) else x
```

Reply:
updated